CMakeLists.txt

```diff
@@ -445,19 +445,18 @@ configure_package_config_file(
 # linkable targets. In this case do not export the targets. Otherwise, just
 # export the slang targets. 
 if(NOT CMAKE_SYSTEM_NAME STREQUAL "Emscripten")
-    if(NOT ${SLANG_BUILD_TYPE} STREQUAL "STATIC")
-        install(
-            EXPORT SlangTargets
+    if(NOT SLANG_BUILD_TYPE STREQUAL "STATIC")
+        export( EXPORT SlangTargets
             FILE ${PROJECT_NAME}Targets.cmake
             NAMESPACE ${PROJECT_NAME}::
-            DESTINATION cmake
```

Comment:
This doesn't look right. `export( )` is for sub-project usage. External installations are meant to use the results from `install(EXPORT )`, like they use other results from `install( )`.